pom.xml

```diff
@@ -116,6 +116,28 @@
                 <extensions>true</extensions>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-invoker-plugin</artifactId>
+                    <version>2.0.0</version>
```

Comment:
@PeteKeller you don't need to repeat the entire configuration from the parent POM here, just provide the version and everything else will be inherited